client/js/libs/handlebars/tz.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
 const moment = require("moment");
+const constants = require("../../constants");
 
 module.exports = function(time) {
-	return moment(time).format("HH:mm");
+	const options = require("../../options");
```

Comment:
No, because otherwise we have a cyclical dependency, with options requiring this file.